### PR TITLE
Lf 2495 supplier displaying for fr pt and sp when editing an existing crop

### PR DESCRIPTION
**Pull request by MwayaB**

### CODE REVIEW

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@MwayaB it looks like some of the changes for your ensemble ticket are on this branch. I think we should either wait for the ensemble feature branch to be merged before merging this PR, or remove those changes from this PR before merging it

**Review by smattingly — APPROVED** [review]

**Review by smattingly — APPROVED** [review]

**Pull request merged**
